toolbox/fixit/main.go

```diff
@@ -94,14 +91,20 @@ func findIssueMetric(repo string, metric *fixItMetric) {
 				}
 			}
 		}
-		// Find the person who labeled the issue.
-		for _, event := range events {
-			if event.GetEvent() == closedState && isFixItWeek(event.GetCreatedAt()) {
-				login := event.GetActor().GetLogin()
-				if login != "istio-merge-robot" {
-					// Not counting the bot
-					metric.issueClosedMap[login]++
-					break
+
+		if issue.GetState() == closedState {
```

Comment:
avoid indentation
if issue.GetState() != closedState {
    continue
}
rest of code follows